osu.Game/Screens/Multi/DrawableRoomPlaylistItem.cs

```diff
@@ -91,6 +96,8 @@ protected override void LoadComplete()
 
         private ScheduledDelegate scheduledRefresh;
 
+        public FillFlowContainer ButtonsContainer { get; private set; }
```

Comment:
Maybe `private` instead?

Reply:
right. i tried to do this another way then backtracked.